### DISCUSSION TITLE
Dreambooth Input tab nowhere to be seen?

### DISCUSSION THREAD

**Discussion by OldSoba** [d8ahazard/sd_dreambooth_extension#1448] [Q&A]
Most of the dreambooth tutorials I see on YouTube have a separate input tab for settings and concepts, and other things but I don't see it in my webui. I am using 1.7.0. I also don't see any Training Wizard (Person) buttons anywhere. Does anyone know why this might be?

<img width="960" alt="Screenshot 2024-01-26 233612" src="https://github.com/d8ahazard/sd_dreambooth_extension/assets/136818168/91165029-858f-40dd-9f27-5e180d3a1c51">
<img width="960" alt="ffwefewfw" src="https://github.com/d8ahazard/sd_dreambooth_extension/assets/136818168/0711aad2-d74d-4595-8097-27d209713903">

**Comment by kallivad** [comment]
Faced with the same issue on 2 different systems. Seems the author didn't test the plugin after every PR.  Couldn't step on his tutorial because of difficulties with interface matching(

**Comment by 192abc123** [comment]
I installed SD also can not install dreamtooth, always said to use the problem, do not know how to solve this problem
